enterprise-architecture/update.py

```diff
@@ -0,0 +1,32 @@
+import csv
+import json
+import ssl
+import urllib.request
+
+
+# Python not detecting certificates on Mac by default - temporary workaround to disable certificate validation
+# https://stackoverflow.com/a/28048260/358804
+ctx = ssl.create_default_context()
+ctx.check_hostname = False
+ctx.verify_mode = ssl.CERT_NONE
```

Comment:
Would love to _not_ do this, but not sure how.

Reply:
@afeld You can reference the GSA root cert while making the request, rather than disable cert validation.

I recommend using the `requests` library, which handles a ton of this stuff for you. You can pass a `ca_file` argument in to the function to use that root store instead of the default one.

Here's an example of that in code GSA and DHS use:

https://github.com/dhs-ncats/pshtt/blob/develop/pshtt/pshtt.py#L153-L202

Reply:
(I think it would be worth doing this before merging the PR.)

Reply:
Sure, but as I [mentioned in Slack](https://gsa-tts.slack.com/archives/C59M1RZC7/p1519921660000564), I don't know where the CA file _is_.